drake/CMakeLists.txt

```diff
@@ -19,9 +19,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 if(WITH_PYTHON_3)
   set(Python_ADDITIONAL_VERSIONS 3.5)
-  find_package(PythonInterp 3 REQUIRED)
+  find_package(PythonInterp 3)
```

Comment:
Note that we've had problems with the PythonInterp before on cygwin.  (see pods.cmake, which has a note about PythonInterp finding the symlink, which caused issues there).  maybe it's ok here?